.travis.yml

```diff
@@ -1,11 +1,16 @@
 language: python
 sudo: false
-python:
-    - 2.7
-    - 3.4
-    - 3.5
-    - 3.6
-    - pypy-5.6.0
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.4
+    - python: 3.5
+    - python: 3.6
+    - python: pypy
+    - python: pypy3
+    - python: 2.7
+      env:
+          - PURE_PYTHON=1
```

Comment:
FWIW you can keep the simpler `python:` list and use `matrix:` to add the special PURE_PYTHON job.  See [zope.testbrowser](https://github.com/zopefoundation/zope.testbrowser/blob/5a54bfb1e7572b8dbc8dc670844112eaf2ab1b75/.travis.yml#L3-L14) for an example.

Reply:
Oh, that's cool. I don't think I knew that. Thanks.